Try to *actually* fix Xwinsock.h on Windows.

pkgw opened the pull request:
Sigh, it helps to get the variable name right and understand why things aren't
working. And I will actually verify that this fix works before deploying this
time.

pkgw committed:
Try to *actually* fix Xwinsock.h on Windows.

Sigh, it helps to get the variable name right and understand why things aren't
working. And I will actually verify that this fix works before deploying this
time.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pkgw committed:
recipe/windows-long64.patch: oh wait, I didn't mess up as much as I thought

Both INT64 *and* LONG64 were problematic. OK, I feel less bad. Still, good to
test before publishing.

pkgw commented:
By the way, the upstream bug is [freedesktop.org #99461](https://bugs.freedesktop.org/show_bug.cgi?id=99461).
